src/routes/PlayQuiz/index.tsx

```diff
@@ -343,6 +344,19 @@ export default function PlayQuiz({ gameId }) {
               <Button size="lg" my="lg" onClick={() => navigate(`/my-quizzes/${userData.userName}`)}>
                 Go to home
               </Button>
+              <Button
+                size="lg"
+                m="lg"
+                variant="gradient"
+                leftIcon={<Icon name="rating" width={24} height={24} />}
+                onClick={() => {
+                  window.open(
+                    `https://docs.google.com/forms/d/e/1FAIpQLSdl3HBQdKbjvI34TqZY-U6UiV4npurnNU_IQZ1OSYksuedU_A/viewform?quizId=${quizInfo.quizId}`,
```

Comment:
We should add this only for quizzes added from the catalog, otherwise is no value in rating a quiz created by the user. We can add an `isAddedFromCatalog` flag to the quiz when it is added from catalog to the local DB. Then only show this button when this flag is true.

Also, we need to pass the quiz name, quiz ID is just a random number now since the quiz is in the local DB for now.

Reply:
Also, can we embed this inside a modal within our app?

Reply:
Make 3 questions only. Modify the language of the last question.